govtool/frontend/src/pages/DRepDirectoryContent.tsx

```diff
@@ -208,27 +208,19 @@ export const DRepDirectoryContent: FC<DRepDirectoryContentProps> = ({
           }}
         >
           {dRepList?.length === 0 && <EmptyStateDrepDirectory />}
-          {dRepListToDisplay?.map((dRep) => {
-            if (
-              isSameDRep(dRep, myDrep?.view) ||
-              isSameDRep(dRep, inProgressDelegation)
```

Comment:
Shouldn't we still exclude delegation in progress since it's displayed at the top of the page?

Reply:
I was wrong, it's not